packages/core/src/styles/components/_grid.scss

```diff
@@ -10,6 +10,16 @@
 		stroke: $ui-03;
 	}
 
+	rect.chart-grid-backdrop-no-stroke {
+		@if $carbon--theme == $carbon--theme--g10 {
+			fill: $ui-01;
+		} @else if $carbon--theme == $carbon--theme--g90 {
+			fill: $carbon--gray-100;
+		} @else {
+			fill: $ui-background;
+		}
+	}
```

Comment:
why do we need this extra classname?

Reply:
The reason why we need `chart-grid-backdrop-no-stroke` classname is because we don't want to apply stroke around the chart when there is no grid line.